docs/compounds/result-card.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
+  <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.critical.css">
+  <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.css">
+</head>
+
+<body>
+  <div class="container">
+    <h1>Result Card</h1>
+    <div class="row">
+      <div class="review-item col-md-4 d-inline-block align-top">
```

Comment:
the width of 1 card for both real result and products on desktop is 3 cols not 4.